cmd/rollout-operator/main.go

```diff
@@ -107,6 +109,17 @@ func main() {
 	ready := atomic.NewBool(false)
 	restart := make(chan string)
 
+	name := os.Getenv("JAEGER_SERVICE_NAME")
```

Comment:
How do you think this would be used? Worth adding to the README?

Reply:
I included this for consistency with how Jaeger is configured in Mimir - very happy to remove it if you think it's not necessary.